examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -523,11 +523,12 @@ func (g *genDeepCopy) doMap(t *types.Type, sw *generator.SnippetWriter) {
 				sw.Do("}\n", nil)
 				sw.Do("(*out)[key] = *newVal\n", nil)
 			} else {
-				sw.Do("if newVal, err := c.DeepCopy(&val); err != nil {\n", nil)
+				sw.Do("newVal, err := c.DeepCopy(&val)\n", nil)
```

Comment:
this produces code that will not compile

see https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/45797/pull-kubernetes-bazel/20933/

```
func DeepCopy_apiextensions_CustomResource(in interface{}, out interface{}, c *conversion.Cloner) error {
	{
		in := in.(*CustomResource)
		out := out.(*CustomResource)
		*out = *in
		newVal, err := c.DeepCopy(&in.ObjectMeta)
		if err != nil {
			return err
		}
		out.ObjectMeta = *newVal.(*v1.ObjectMeta)

		newVal, err := c.DeepCopy(&in.Spec)
		if err != nil {
			return err
		}
		out.Spec = *newVal.(*CustomResourceSpec)

		newVal, err := c.DeepCopy(&in.Status)
		if err != nil {
			return err
		}
		out.Status = *newVal.(*CustomResourceStatus)

		return nil
	}
}
```